helm_deploy/values-dev.yaml

```diff
@@ -44,4 +44,4 @@ env:
   TOKENVERIFICATION_API_URL: https://token-verification-api-dev.prison.service.justice.gov.uk
   TOKENVERIFICATION_API_ENABLED: true
   FEEDBACK_SUPPORT_URL: "https://support-dev.hmpps.service.justice.gov.uk/feedback-and-support"
-  LICENCE_RISK_MANAGEMENT_VERSION: 2
+  LICENCE_RISK_MANAGEMENT_VERSION: '2'
```

Comment:
This is fine, but these are equivalent - env vars are passed to running service as strings